src/components/Breadcrumbs/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import Breadcrumbs from "react-router-dynamic-breadcrumbs"
+import { Link, BrowserRouter as Router } from "react-router-dom"
+import PropTypes from "prop-types"
+
+const routesList = {
+  "/": "Home",
+  "/login": "Login",
+  "/register": "Register",
+  "/visits": "Visits",
+  "/visits/details/test-id": "View a single Visit",
+  "/visits/create-new": "New Visit",
+  "/visits/edit/test-id": "Edit a Visit",
+  "/visits": "All Visits",
+  "/patients": "Patients",
+  "/patients/details/test-id": "Patient Details",
+  "/patients/create-new": "Patient Create New",
+  "/patients/edit/test-id": "Patient Edit"
+}
+
+function BreadcrumbsComponent() {
+  return (
+    <Router>
```

Comment:
We are not supposed to have two Router components in our app. There should be only one and that one already exists in [App.js](https://github.com/SocialHackerCreteClass/FrodidApp-FrontEnd/blob/develop/src/App.js) file.
I think it is safe to delete the Router and render just the Breadcrumbs here